Resources/Textures/Objects/Fun/bwoink_hammer.rsi/meta.json

```diff
@@ -0,0 +1,22 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "taken from BigIron github",
```

Comment:
plz give repo link and commit hash
Link to the file at the commit also works

Reply:
oh shit I forgot to edit this.